server/api_server/schemas.py

```
@@ -0,0 +1,621 @@
+    "app.php_errors.log", "execution_queue.log", "db_is_locked.log"
+]
+
+
+def validate_mac(value: str) -> str:
+    """Validate and normalize MAC address format."""
+    # Allow "Internet" as a special case for the gateway/WAN device
+    if value == "Internet":
+        return value
+
+    if not re.match(MAC_PATTERN, value):
+        raise ValueError(f"Invalid MAC address format: {value}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)